hudi-sync/hudi-dla-sync/src/main/java/org/apache/hudi/dla/DLASyncConfig.java

```diff
@@ -62,6 +62,9 @@
   @Parameter(names = {"--skip-ro-suffix"}, description = "Skip the `_ro` suffix for Read optimized table, when registering")
   public Boolean skipROSuffix = false;
 
+  @Parameter(names = {"--skip-rt-sync"}, description = "Skip the RT table syncing")
+  public Boolean skipRTSync = true;
```

Comment:
just one more comment, here the default value to be false would be better, since some users have synced both ro and rt tables to dla.

Reply:
done